shell/src/main/java/alluxio/cli/fs/FileSystemShell.java

```diff
@@ -36,11 +36,7 @@
 public final class FileSystemShell extends AbstractShell {
   private static final Logger LOG = LoggerFactory.getLogger(FileSystemShell.class);
 
-  private static final Map<String, String[]> CMD_ALIAS = ImmutableMap.<String, String[]>builder()
-      .put("lsr", new String[] {"ls", "-R"})
-      .put("rmr", new String[] {"rm", "-R"})
-      .put("umount", new String[] {"unmount"})
-      .build();
+  private static final Map<String, String[]> CMD_ALIAS = ImmutableMap.<String, String[]>builder().build();
```

Comment:
we should keep `umount` as an alias of `unmount`

Reply:
why?

Reply:
command aliases mark warn the user that the use of the alias is deprecated. It should be removed if that is the case. Otherwise we should have another mechanism for handling non-deprecated aliases.

Reply:
Previously, Alluxio fs supports `unmount` only and Alluxio fuse supports `umount` only, which to me is confusing. So I actually added this alias recently to make it easier for users so they don't need to remember if it is `umount` or `unmount` --- we will support both.

Reply:
I see. Agree to have a separate mechanism (without showing deprecation warnings). Up to you if you want to solve it in this PR or separate PR

Reply:
We should unify this then. Either a user uses `unmount` or `umount` for all Alluxio related operations. Is there a point to having both?

Reply:
In Unix/Linux world  it is `umount` for historical reasons as a abbreviated version of `unmount`. However, Alluxio has been supporting `unmount` but not `umount` for years since very early releases. I don't see any benefits to require users to remember Alluxio supports only `umount` or `unmount`.

Reply:
I think I'm just trying to get out of the extra work here 😅 

I'll implement something for non-deprecated aliases in this PR.